torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2037,24 +2037,6 @@ RegisterOperators reg2({
           return 0;
         }),
 
-    Operator(
```

Comment:
Why do you get to remove this?

Reply:
`DEFINE_BINARY_OPS` already emit the int/float mixed ops (called `DEFINE_INT_FLOAT_OP` [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/register_prim_ops.cpp#L1062)), this is not necessary anymore. This was due to some merge conflicts that we did not aware of before..